ntbs-service/Services/DrugResistanceProfileService.cs

```diff
@@ -14,19 +14,19 @@ public class DrugResistanceProfileService : IDrugResistanceProfilesService
     {
         private readonly INotificationService _notificationService;
         private readonly INotificationRepository _notificationRepository;
-        private readonly IMdrService _mdrService;
+        private readonly IEnhancedSurveillanceAlertsService EnhancedSurveillanceAlertsService;
```

Comment:
Rider suggested name here does not follow the convention we're using (and I do prefer the naming we're using - but i'm happy to be crowd-nudged away from it).